csharp/AppEncryption/.editorconfig

```diff
@@ -0,0 +1,26 @@
+root = true
```

Comment:
These copies shouldn't be needed as the existing `.editorconfig` file located in the repo's root will be discovered by EditorConfig plugins.

From EditorConfig documentation:
> When opening a file, EditorConfig plugins look for a file named .editorconfig in the directory of the opened file and in every parent directory. A search for .editorconfig files will stop if the root filepath is reached or an EditorConfig file with root=true is found.

Reply:
See, I don't get why I wasn't getting .editorconfig behavior out of Visual Studio then.  It wasn't trimming whitespace etc.  I wonder if their implementation stops at the solution directory or something.

Reply:
Hmm, according to the Visual Studio docs, as long the setting hasn't been disabled their implementation should be searching parent directories as well, see [Troubleshoot EditorConfig settings](https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2019#troubleshoot-editorconfig-settings)

Reply:
Yeah, I'll back this out and investigate.